src/diff.sh

```diff
@@ -90,6 +95,32 @@ function diff_parser_options()
 
 }
 
+# Show the diff result between folders in two columns equally divided.
+#
+# @folder_1 Path to the first folder
+# @folder_2 Path to the second folder
+# @flag How to display a command, see `src/kwlib.sh` function `cmd_manager`
+function diff_folders()
+{
+  local folder_1="$1"
+  local folder_2="$2"
+  local flag="$3"
+  local diff_cmd
+  local columns
+
+  diff_cmd="diff --color=always -rq"
+  flag=${flag:-''}
+
+  diff_cmd="$diff_cmd $folder_1 $folder_2 | grep $folder_1"
+
+  if [[ ! -d "$folder_1" || ! -d "$folder_2" ]]; then
+    complain "Make sure that $folder_1 and $folder_2 are valid folders"
```

Comment:
For string concatenation:
https://kworkflow.org/content/codingstyle.html#string-concatenation